docroot/modules/custom/va_gov_build_trigger/src/Plugin/VAGov/Environment/BRD.php

```diff
@@ -199,4 +199,21 @@ public function getBuildTriggerFormClass() : string {
     return BrdBuildTriggerForm::class;
   }
 
+  /**
+   * Gets the current Jenkins API token.
+   *
+   * @return string
+   *   The value of the value of ssm param named
+   *   '/cms/va-cms-bot/jenkins-api-token', or '' if not found.
+   */
+  private function getJenkinsApiToken() {
+    $cmd = 'aws ssm get-parameter --name "/cms/va-cms-bot/jenkins-api-token" --with-decryption --filter Parameter.Value';
```

Comment:
```suggestion
    $cmd = 'aws ssm get-parameter --name "/cms/va-cms-bot/jenkins-api-token" --with-decryption --query Parameter.Value';
```

Should filter be query here?

Reply:
We also might as well add 

```
--output text
```

as that will get rid of double quotes surrounding the value.